libff/algebra/fields/tests/test_fields.cpp

```diff
@@ -7,6 +7,7 @@
 
 #include <gtest/gtest.h>
 #include <libff/algebra/curves/bls12_377/bls12_377_pp.hpp>
+#include <libff/algebra/curves/bls12_381/bls12_381_pp.hpp> // VV
```

Comment:
Please make sure to remove all these `// VV` comments @vesselinux (do a global search to remove any remaining ones)

Reply:
Done. Thanks @dtebbs for the useful command! Indeed two of the results are from legit variables. Another two refer to the signed digits fix and I planned to remove them after successfully merging PR #71 . One VV was still left by mistake.